lib/pages/error_page/error_page_style.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+class ErrorPageStyle {
+  static ColorFilter backgroundColorFilter(BuildContext context) =>
+      ColorFilter.mode(
+        Theme.of(context).colorScheme.primaryContainer,
+        BlendMode.srcATop,
+      );
+
+  static const double backgroundIconWidthWeb = 448.0;
+  static const double backgroundIconWidthMobile = 280.0;
+
+  static const EdgeInsets textPadding = EdgeInsets.symmetric(vertical: 32.0);
+
+  static TextStyle? titleTextStyle(BuildContext context) =>
+      Theme.of(context).textTheme.headlineSmall!.copyWith(
```

Comment:
<img width="1290" alt="Screenshot 2023-09-18 at 00 54 49" src="https://github.com/linagora/twake-on-matrix/assets/99852347/85845ea8-3c0e-460c-b1b9-9c5e220fadac">

M3/headline/large not headlineSmall

Reply:
didn't see it was different styles for web and mobile